presto-docs/src/main/sphinx/release/release-328.rst

```diff
@@ -0,0 +1,74 @@
+========================
+Release 328 (8 Jan 2020)
+========================
+
+General Changes
+---------------
+
+* Fix correctness issue for certain correlated join queries when the correlated subquery on
+  the right produces no rows. (:issue:`1969`)
+* Fix incorrect handling of multi-byte characters for :doc:`/functions/regexp` when
+  the pattern is empty. (:issue:`2313`)
+* Fix failure when join criteria contains columns of different types. (:issue:`2320`)
+* Fix failure for complex outer join queries when dynamic filtering is enabled. (:issue:`2363`)
+* Improve support for correlated queries. (:issue:`1969`)
+* Allow inserting values of a larger type into as smaller type when the values fit. For example,
+  ``BIGINT`` into ``SMALLINT``, or ``VARCHAR(10)`` into ``VARCHAR(3)``. Values that don't fit will
+  cause an error at runtime. (:issue:`2061`)
+* Add :func:``regexp_count`` and :func:``regexp_position`` functions. (:issue:`2136`)
+
+Security Changes
+----------------
+
+* Fix a security issue allowing users to gain unauthorized access to Presto cluster
+  when using password authenticator with LDAP. (:issue:`2356`)
+* Add support for LDAP referrals in LDAP password authenticator. (:issue:`2354`)
+
+JDBC Driver
+-----------
+
+* Fix behavior of ``java.sql.Connection#commit()`` and ``java.sql.Connection#rollback()``
+  methods when no statements performed in a transaction. Previously, these methods
+  would fail. (:issue:`2339`)
+* Fix failure when restoring autocommit mode with
+  ``java.sql.Connection#setAutocommit()`` (:issue:`2338`)
+
+Hive Connector Changes
+----------------------
+
+* Reduce query latency and Hive metastore load when ``optimizer.join-reordering-strategy``
```

Comment:
Let's change this to
```
when using the ``AUTOMATIC`` join reordering strategy
```
Since this is controlled by both a config and session property